### PR TITLE
Add Point2d message, and use it in Pose2D

### LINKED ISSUE #66: Suggestion for new msg type: vision_msgs/Point
Along with the changes in https://github.com/ros-perception/vision_msgs/pull/64, I want to suggest separating the position (x, y) from `vision_msgs/Pose` into a new msg `vision_msgs/Position`.

Often, there are cases where I want to store a point in an image (where heading doesn't make sense), and there is no appropriate message type for it. vision_msgs seems like the appropriate place for this sort of msg.

This is similar to how `position` is a field in [geometry_msgs/Pose.msg](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Pose.msg)

### PR DESCRIPTION
**Pull request by ijnek**
Resolves: #66

### CODE REVIEW

**Review by SteveMacenski — APPROVED** [review]
@Kukanani ?

File: msg/Point2D.msg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Represents a 2D point (coordinates).
```
Comment (Kukanani) [inline msg/Point2D.msg]:
```suggestion
# Represents a 2D point in pixel coordinates.
```

**Review by Kukanani — APPROVED** [review]
Looks good; let's call out our units explicitly

**Comment by Kukanani** [comment]
Thanks for the code :)

**Comment by ijnek** [comment]
@Kukanani Would you be able to make a rolling release with the new changes when you've got the time?

**Comment by Kukanani** [comment]
https://github.com/ros/rosdistro/pull/32485

**Comment by ijnek** [comment]
Thanks @Kukanani

**Pull request merged**